AI/Nullkiller/CMakeLists.txt

```diff
@@ -157,7 +157,9 @@ else()
 endif()
 
 target_include_directories(Nullkiller PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(Nullkiller PUBLIC vcmi fuzzylite::fuzzylite)
+if (NOT EMSCRIPTEN)
+    target_link_libraries(Nullkiller PUBLIC vcmi fuzzylite::fuzzylite TBB::tbb)
```

Comment:
tbb was not here before

Reply:
True, it was in the previous version. I will fix it.

Reply:
does NKAI no longer use tbb? As I brought this dep back in beta.